vendor/cel.dev/expr/MODULE.bazel

```diff
@@ -8,7 +8,7 @@ bazel_dep(
 )
 bazel_dep(
     name = "gazelle",
-    version = "0.36.0",
+    version = "0.41.0",
```

Comment:
@ksimon1 is this a change we want to have, or should we exclude the bazel plugin in the config?

Reply:
This is indirect dependency - https://github.com/kubevirt/ssp-operator/pull/1109/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R43
If we exclude it, some dependency (e.g. github.com/google/cel-go) might not be updated